### PR TITLE
Update lambda.md

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/lambda.md [hunk 1]
```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
 
 ### Collect Amazon CloudWatch Metrics
 
-To collect Amazon CloudWatch Metrics, see [Amazon CloudWatch Source For Metrics.](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
-
-AWS Namespace tag to filter in source for Lambda will be - **AWS/Lambda**
-
-* **Metadata: **Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+* Sumo Logic supports collecting metrics using two source types
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:171]:
```suggestion
* Sumo Logic supports collecting metrics using two source types:
```

File: docs/integrations/amazon-aws/lambda.md [hunk 2]
```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
...
+    * Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
+       or
+    * Configure an [AWS Kinesis Firehose for Metrics Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS_Kinesis_Firehose_for_Metrics_Source) (Recommended)
+:::note
+Namespace for **Amazon Lambda** Service is **AWS/Lambda**.
+:::
+    * **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account” field.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:178]:
```suggestion
    * **Metadata**. Add an **account** field to the source and assign it a value that is a friendly name/alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. The **account** field allows you to query metrics.
```

Comment (JainM6) [reply docs/integrations/amazon-aws/lambda.md:178]: Can you please check how the "account" field is written in UI? If it's in uppercase, then we should change here as well.

File: docs/integrations/amazon-aws/lambda.md [hunk 3]
```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
...
 
-To collect Amazon CloudWatch Metrics, see [Amazon CloudWatch Source For Metrics.](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
-
-AWS Namespace tag to filter in source for Lambda will be - **AWS/Lambda**
-
-* **Metadata: **Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+* Sumo Logic supports collecting metrics using two source types:
+    * Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
+       or
+    * Configure an [AWS Kinesis Firehose for Metrics Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS_Kinesis_Firehose_for_Metrics_Source) (Recommended)
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:174]:
```suggestion
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)
```

File: docs/integrations/amazon-aws/lambda.md (hunk 2 above)
Comment (nmohantyctr) [reply docs/integrations/amazon-aws/lambda.md:178]: Its mentioned as "account" itself.

Comment (JainM6) [reply docs/integrations/amazon-aws/lambda.md:178]: @nmohantyctr Thanks for the confirmation.

File: docs/integrations/amazon-aws/lambda.md [hunk 4]
```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
 
 ### Collect Amazon CloudWatch Metrics
 
-To collect Amazon CloudWatch Metrics, see [Amazon CloudWatch Source For Metrics.](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
-
-AWS Namespace tag to filter in source for Lambda will be - **AWS/Lambda**
-
-* **Metadata: **Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+* Sumo Logic supports collecting metrics using two source types:
```
Comment (JainM6) [inline docs/integrations/amazon-aws/lambda.md:171]: I see we have mentioned the same sentence twice, one is with the link and another one is without the link. Why do we have to write it twice?
Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
       or
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)

**Review by JainM6 — APPROVED** [review]

**Pull request merged**